core/src/main/java/org/infinispan/health/HostInfo.java

```diff
@@ -0,0 +1,25 @@
+package org.infinispan.health;
+
+/**
+ * Information about the host.
+ *
+ * @author Sebastian Łaskawiec
+ * @since 9.0
+ */
+public interface HostInfo {
```

Comment:
I mean, the info below is accessible via JMX mbeans provided by the JVM itself (see [here](https://docs.oracle.com/javase/8/docs/api/java/lang/management/ManagementFactory.html)). Not sure why we should provide a facade for those?